src/update_feed.py

```diff
@@ -405,31 +614,45 @@ def process_assets(
         "application/x-x509-ca-cert",
     ]
 
-    for asset in assets:
-        # early filter for irrelevant assets
-        if asset.content_type in skip_content_types:
-            continue
+    with ThreadPoolExecutor() as executor:
```

Comment:
This is so tedious compared to goroutines 😅 I can understand why you'd want to rewrite this.

Reply:
😅